### ISSUE TITLE
Switch to short folder names and module IDs

### ISSUE THREAD

**Issue by reckart** [dkpro/dkpro-c4corpus#5]
Switch module folder names and module artifactIDs from e.g.

```
de.tudarmstadt.ukp.dkpro.c4corpus.boilerplate
```

to

```
dkpro-c4corpus-boilerplate
```

**Comment by reckart** [comment]
Looks like now all the folders are double?

**Comment by habernal** [comment]
Refactoring went crazy...

**Comment by habernal** [comment]
Fixed by df75784c64aee36296aa1643863d3d1561adb11d
